contracts/src/v0.8/SimpleWriteAccessController.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./dev/ConfirmedOwner.sol";
```

Comment:
I'm noticing that the latest `ConfirmedOwner` code is actually in v0.7, where a few extra contracts have been to make ownership more configurable.

This might be beyond the scope of this PR (open to opinions), but could we migrate the v0.7 ConfirmedOwner and associated contracts over to v0.8 too? If not, we should make this a separate ticket so that it's not forgotten.

Reply:
I'll include that in this PR, no problem

Reply:
As it depends on a `/dev` contract now, should these contracts go under `/dev` as well?

Reply:
Good question. I believe all contracts associated with `Operator` and `OperatorForwarder` have been audited, so they're ready to be moved out of `dev/` and we can safely keep these where they are.

Reply:
Cool! I'll move `ConfirmedOwner` and `ConfirmedOwnerWithProposal` out of `/dev`. `Operator` and related for another PR better? 😄

Reply:
Sounds good! I'll open another PR for the Operator stuff :)